cloudprem/physical/modules/vpn/endpoint.tf

```diff
@@ -0,0 +1,40 @@
+# AWS vpn client endpoint
+resource "aws_ec2_client_vpn_endpoint" "vpn-client" {
+  description            = "${local.identifier}-vpn-client"
+  server_certificate_arn = aws_acm_certificate.server.arn
+  #  vpc_id                 = var.vpc_id
+  #  security_group_ids     = [aws_security_group.vpn.id]
+  client_cidr_block = var.client_cidr_block
+  #  session_timeout_hours  = var.session_timeout_hours
```

Comment:
Can we clean up these comments? Months from now I won't know if these are left-over from testing or placeholders or... :)

Reply:
Yep. They should've been cleaned up, good catch.